gatsby/src/pages/drugs/{BnfDrug.slug}/index.tsx

```diff
@@ -141,37 +163,39 @@ const DrugPage: FC<DrugPageProps> = ({ data: { bnfDrug } }) => {
 
 			<PageHeader
 				id="content-start"
-				heading={<span dangerouslySetInnerHTML={{ __html: bnfDrug.title }} />}
+				heading={<span dangerouslySetInnerHTML={{ __html: title }} />}
 			/>
 
-			<Grid gutter="loose">
-				<GridItem cols={12} md={8} lg={9} className="hide-print">
+			<div className={styles.contentWrapper}>
+				<div className={styles.sectionNav}>
 					<SectionNav
 						sections={orderedSections.map(({ potName, slug }) => ({
 							id: slug,
 							title: potName,
 						}))}
 					/>
-				</GridItem>
-				<GridItem cols={12} md={4} lg={3} className="hide-print">
-					<Panel>Quick links will go here</Panel>
-				</GridItem>
-				<GridItem cols={12} md={8} lg={9}>
+				</div>
+				<div className={styles.aside}>
+					<Panel>
+						<h2 className="h5">Medicinal forms and&nbsp;pricing</h2>
```

Comment:
Really minor one - is the nbsp intentional?

Reply:
It is. It keep the `and` and `pricing` together to avoid orphaned words when it wraps (e.g. mobile)